docs/serverless-demo.md

```diff
@@ -69,40 +67,44 @@ mutation PausePlaybackMutation {
 
 You can begin [identifying your client traffic](https://www.apollographql.com/docs/graphos/metrics/client-awareness) by ensuring the proper headers are sent. GraphOS will automatically associate the graph usage with the identified client information:
 
-| Header name                    | Header value                                                                             |
-| ------------------------------ | ---------------------------------------------------------------------------------------- |
-| `apollographql-client-name`    | The name of your integration.                                                            |
-| `apollographql-client-version` | Increment this value with version numbers for your integration as its API usage changes. |
+| Header name                    | Header value                                                                        |
+| ------------------------------ | ----------------------------------------------------------------------------------- |
+| `apollographql-client-name`    | The name of your client.                                                            |
+| `apollographql-client-version` | Increment this value with version numbers for your client as its API usage changes. |
```

Comment:
Possibly add something about this being enterprise-only?

Reply:
great catch, we'll remove this